testsuite/features/build_validation/core/allcli_sanity.feature

```diff
@@ -34,285 +34,285 @@ Feature: Sanity checks
   Scenario: The proxy is healthy
     Then "proxy" should have a FQDN
     And reverse resolution should work for "proxy"
-    And "proxy" should communicate with the server
+    And "proxy" should communicate with the server using "eth0"
```

Comment:
a bit repetitive, isn't it?

Reply:
Well, what I can't do is to let the "should communicate" without the parameter, and the "should not communicate" without the parameter, I prefer to have the same format in both positive and negative cases.

Reply:
I would remove the parameter in both cases.